### ISSUE TITLE
Flutter compile times on iOS have regressed

### ISSUE THREAD

**Issue by Hixie** [dart-lang/sdk#35071]
![screen shot 2018-11-06 at 9 29 39 am](https://user-images.githubusercontent.com/551196/48082124-91b24400-e1a6-11e8-9174-6721c5fddf27.png)

**Comment by Hixie** [comment]
Similarly, here's the long-term history of hello_world_ios__compile debug_full_compile_millis:
![screen shot 2018-11-06 at 9 30 57 am](https://user-images.githubusercontent.com/551196/48086410-a811cd00-e1b1-11e8-8dfa-da46cb538362.png)

**Comment by Hixie** [comment]
cc @cbracken @a-siva

**Comment by aam** [comment]
Since this shows up on ios only -  could it be related to certificate handling/signing done by xcode?

**Comment by mit-mit** [comment]
Do we have a benchmark for `flutter build aot` so we can compare?

**Comment by mraleph** [comment]
Yes we do - benchmark seems to confirm that we regressed. Seems to be related to how much we spend in the front-end. 

![rwo81oaklnf](https://user-images.githubusercontent.com/131846/48141233-bf80b100-e2aa-11e8-8582-5657a4e0928b.png)
